package.json

```diff
@@ -26,9 +26,9 @@
     "lint": "aegir lint",
     "release": "aegir release",
     "build": "aegir build",
-    "test": "aegir test",
+    "test": "npm run test:node && npm run test:browser",
     "test:node": "aegir test --target node",
-    "test:browser": "aegir test --target browser",
+    "test:browser": "aegir test --target browser --files test/**/*.browser.spec.{js,cjs,mjs}",
```

Comment:
This needs to happen because of: https://github.com/ipfs/aegir/issues/1097 Also, the test files had to be renamed because of this.